README.md

```diff
@@ -2,3 +2,8 @@
 
 client utils for https://github.com/brave/bitgo-proxy. tests are run in that
 repo.
+
+# setup
+
+BitGo does not yet support Node 12+, so run `nvm use lts/dubnium` to use
```

Comment:
FWIW, we use Node v10 LTS in ethereum-remote-client via `nvm use 10` which is slightly less typing :)